web/js/deerInitializer.js

```diff
@@ -163,39 +195,302 @@ DEER.TEMPLATES.personMulti = function(obj, options = {}) {
+        })
+        let practicesHTML = `
+            <h4>Practices at Experience "${UTILS.getLabel(experienceData)}"</h4>
+            <p>Practices you add will appear here and can be removed, but not edited.</p>
+            <ul id="practicesInExperience">
+                ${relatedPracticesByName}
+            </ul>
+        `
+        
+        //Gather relatedSenses, an array of URIs
+        let relatedSensesByName = ``
+        //experienceData.relatedSenses is probably a Set or List of String URIs, we want their label
+        relatedSenses.items.forEach((val)=>{
+            let name = ""
+            if(typeof val === "object"){
+                //See if the value is the URI we want
+                let itemURI = UTILS.getValue(val)
+                if(itemURI.indexOf("http://") > -1 || itemURI.indexOf("https://") > -1){
+                    name = `
+                    <li>
+                        <deer-view deer-id="${itemURI}" deer-template="mostUpToLabelHelper"></deer-view>
+                        <a class="tag is-rounded is-small text-error" onclick="LR.utils.disassociateObject(event, '${itemURI}', '${experienceData["@id"]}')">Remove</a>
+                    </li>
+                    `
+                }
+                else{
+                    //We know it is just a string of some kind.  Just use it.
+                    //TODO what should we do here?
+                    name = `
+                    <li>
+                        ${itemURI}
+                    </li>
+                    `
+                }
+            }
+            else{
+                if(val.indexOf("http://") > -1 || val.indexOf("https://") > -1){
+                    //We expect this is item entry is the URI we were looking for
+                    name = `
+                    <li>
+                        <deer-view deer-id="${val}" deer-template="label"></deer-view>
+                        <a class="tag is-rounded is-small text-error" onclick="LR.utils.disassociateObject(event, '${val}', '${experienceData["@id"]}')">Remove</a>
+                    </li>
+                    `
+                }
+                else{
+                    //We know it is just a string of some kind and not the URI, so we can show this string.  
+                    //TODO what should we do here?
+                    name = `
+                    <li>
+                        ${val}
+                    </li>
+                    `
+                }
+            }
+            relatedSensesByName += name
+        })
+        let sensesHTML = `
+            <h4>Senses at Experience "${UTILS.getLabel(experienceData)}"</h4>
+            <p>Senses you add will appear here and can be removed, but not edited.</p>
+            <ul id="sensesInExperience">
+                ${relatedSensesByName}
+            </ul>
+        `     
+        let researchersHTML = `<dt>LRDA Researchers Involved</dt><dd><ul id="researchersInExperience">${contributorsByName}</ul></dd>`

```

Comment:
I won't make you do anything, but this naming and immediate use of the template literal is a bit of an anti-pattern. They are useful for their multi-line templates.

```
html = `<header>Htext</header>
<content>
    lorem
</content>
<footer>contact us</footer>`
```

Looks just like you expect.

Reply:
There will be a stage where I go over all the templates and smooth these kinds of things out.  Since I am trying to use DEER as much as possible, a lot of things are still broken out goofy to track through DEER better.